src/Components/Timer/Timer.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from 'react'
+import './Timer.scss'
+
+interface Props {
+  total_seconds: number,
+  stopTimer: (arg0: string) => {}
+}
+
+const Timer: React.FC<Props> = ({ total_seconds, stopTimer }) => {
+  const [ elapsedTime, setElapsedTime ] = useState(0)
+  const [ inProgress, toggleInProgress ] = useState(false)
+  let colorClass: string = 'green'
```

Comment:
Did typescript error out when you didn't type the variable colorClass (colorClass: string) ? I'm just curious!